example/example_variant.py

```
@@ -0,0 +1,119 @@
+                         xl=np.array(VARIABLE_BOUNDS[0][start_idx:end_idx]),
+                         xu=np.array(VARIABLE_BOUNDS[1][start_idx:end_idx])
+                         )
+        
+    def _evaluate(self, x, out, *args, **kwargs) -> None:
+        """ Evaluate the problem """
+
+        # Create variant benchmark object and evaluate the design vector(s)
+        benchmark = AircraftVariant(design_vars=x,
+                                    variant_index=VARIANT_TYPE)
+        
+        objectives, constraints, combined_CV = benchmark.evaluate()
```

Ruff (0.15.6)

[warning] Unpacked variable `combined_CV` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)